src/main.cpp

```diff
@@ -179,6 +201,14 @@ void slice(int argc, char **argv)
                     case 'v':
                         cura::increaseVerboseLevel();
                         break;
+#ifdef _OPENMP
+                    case 'm':
+                        str++;
+                        n_threads = *str - '0';
+                        n_threads = std::max(1, n_threads);
+                        omp_set_num_threads(n_threads);
+                        break;
```

Comment:
I can't really lift the `case` and the `break` and it would be kind of weird to have a function to change the local variable `str`, so then we would introduce a new function to remove the code duplication of 3 lines.
Seems a bit useless.